src/routes/+error.svelte

```
@@ -0,0 +1,158 @@
+<script>
+	import { page } from '$app/stores';
+	import { _, t } from 'svelte-i18n';
```

ESLint

[error] 't' is defined but never used.

(no-unused-vars)